MtfhReportingDataListener/UseCase/TenureUseCase.cs

```diff
@@ -58,11 +54,14 @@ public async Task ProcessMessageAsync(EntityEventSns message)
                 },
                 Tenure = tenure
             };
-            var record = BuildTenureRecord(schema.Schema, tenureChangeEvent);
 
             var topicName = Environment.GetEnvironmentVariable("TENURE_SCHEMA_NAME");
+            var schemaRegistryUrl = Environment.GetEnvironmentVariable("KAFKA_SCHEMA_REGISTRY_HOSTNAME");
```

Comment:
If the schema registry url is the same for all use cases, can we move this line in the Gateway method so we don't have to repeat it for each usecase?